src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -232,21 +281,30 @@ public virtual string CreateToken(string payload, SigningCredentials signingCred
 
         private string CreateTokenPrivate(JObject payload, SigningCredentials signingCredentials, EncryptingCredentials encryptingCredentials, string algorithm)
```

Comment:
Can we rename the parameter 'algorithm' to 'compressionAlgorithm'?

Reply:
Made this change. 👍 

I also went ahead and changed the name to `compressionAlgorithm` in all the other private methods as well.

I kept `public string EncryptToken(string innerJwt, EncryptingCredentials encryptingCredentials, string algorithm)` as is since it's a public method and changing the parameter name is a breaking change.